include/wx/spinbutt.h

```diff
@@ -61,12 +61,14 @@ class WXDLLIMPEXP_CORE wxSpinButtonBase : public wxControl
 
     // is this spin button vertically oriented?
     bool IsVertical() const { return (m_windowStyle & wxSP_VERTICAL) != 0; }
+    virtual void SetIncrement(int WXUNUSED(value)) { m_increment = 1; }
+    virtual int GetIncrement() const { return m_increment; }
```

Comment:
This doesn't make much sense to me, what's the point of having `m_increment` but always returning 1? The simplest alternative is to do it like this:
```suggestion
    virtual void SetIncrement(int WXUNUSED(value)) { }
    virtual int GetIncrement() const { return 1; }
```